src/coreclr/tools/Common/Compiler/Win32Resources/ResourceData.Reader.cs

```diff
@@ -5,7 +5,11 @@
 using System.Reflection.Metadata;
 using System.Reflection.PortableExecutable;
 
+#if HOST_MODEL
+namespace Microsoft.NET.HostModel.Win32Resources
```

Comment:
Can you please file an issue to move this code out of the "Compiler" directore and change the namespace such that we don't need the ifdefs everywhere.
I'm totally fine with it for this PR (let's not make it even bigger), but I personally don't like this, especially since it's in the same repo.

Reply:
I filed #90063